index.html

```diff
@@ -142,7 +144,7 @@ <h2>Icons</h2>
             <li><i class="fab fa-github"></i><span>.fa-github</span></li>
```

Comment:
maybe we can add some text at the beginning of this icons showing the CDN link tag, so if someone wants to use the styleguide standalone files they know they need to add a cdn for the icons (maybe in the README too?)

Reply:
Good idea. I'll create a new story to work with that.